Formula/mpich.rb

```diff
@@ -39,15 +48,26 @@ def install
     end
 
     system "./configure", "--disable-dependency-tracking",
+                          "--enable-fast=all,O3",
+                          "--enable-g=dbg",
+                          "--enable-romio",
+                          "--enable-shared",
+                          "--enable-sharedlibs=gcc-osx",
+                          "--with-pm=hydra",
+                          "CC=gcc-"+Formula["gcc"].any_installed_version.major,
+                          "CXX=g++-"+Formula["gcc"].any_installed_version.major,
+                          "FC=gfortran-"+Formula["gcc"].any_installed_version.major,
+                          "F77=gfortran-"+Formula["gcc"].any_installed_version.major,
```

Comment:
```suggestion
                          "CC=gcc-#{Formula["gcc"].any_installed_version.major}",
                          "CXX=g++-#{Formula["gcc"].any_installed_version.major}",
                          "FC=gfortran-#{Formula["gcc"].any_installed_version.major}",
                          "F77=gfortran-#{Formula["gcc"].any_installed_version.major}",
```
Most other formulae use interpolation rather than concatenation, so this should make things a bit more consistent, I think.

Reply:
Oh, and squash your commits together for any changes you make, please. Thanks!

Reply:
I made the changes you suggested and I believe I squashed everything in a single commit but I am seriously git-impaired, so apologies if I messed up

Reply:
You did great; thanks very much!